src/pages-conditional/what-is-ethereum.tsx

```diff
@@ -917,7 +921,7 @@ const WhatIsEthereumPage = ({
                 . There are currently over 1777 individual nodes running
                 Ethereum network. This makes Ethereum one of the most
                 decentralized cryptocurrencies out there, second only to
-                bitcoin.
+                Bitcoin.
```

Comment:
Do we mean cryptocurrencies or blockchains here? (i.e. ether & bitcoin or Ethereum & Bitcoin?)